libcxx/test/std/utilities/function.objects/func.wrap/func.wrap.func/func.wrap.func.con/deduct_F.verify.cpp

```diff
@@ -18,6 +18,7 @@
 // Make sure we stick to the specification.
```

Comment:
I think we should replace the current test by the SFINAE test you added. We normally don't test constraints using clang-verify tests, we need to use SFINAE tests. So in that sense I think the original test wasn't written the way it should have.